build and debug on current MacOS, test DSN will now also apply parameters (#109)
openlink/iODBC: README_MACOSX.md

Then, execute the following commands in a terminal session, to build all the frameworks and demo applications:

```shell
cd mac 
make
```

> In case there are errors first try to clean up temporary files:

Reviewer: ```suggestion
If there are errors, first try to clean up temporary files:
```

```shell
sudo make clean
```

After building the iODBC libraries and applications, you have to install them on your system with the command:

```shell
sudo make install
```

> For generating and installing a development build with debug symbols run:

Reviewer: ```suggestion
To generate and install a development build with debug symbols, run:
```
